packages/governance/src/paramGovernance/paramManager.js

```diff
@@ -266,9 +265,9 @@ const makeParamManagerBuilder = zoe => {
   const getParams = () => {
     /** @type {Record<Keyword,ParamDescription>} */
     const descriptions = {};
-    namesToParams.entries().forEach(([name, param]) => {
+    for (const [name, param] of namesToParams.entries()) {
```

Comment:
How does this improve the code?

Reply:
Now that `entries()` returns an Iterable, that return value no longer has a `.forEach` method, so some rewrite was necessary regardless. For a `.forEach` specifically, it seems most painless to turn it into a `for` loop.